test/functional/command_write_concern_tests.js

```diff
@@ -718,17 +722,19 @@ describe('Command Write Concern', function() {
             test.equal(null, err);
             var db = client.db(configuration.db);
 
-            db.collection('test').dropIndexes({
-              w: 2,
-              wtimeout: 1000
-            },
-            function(err) {
-              test.equal(null, err);
-              test.deepEqual({ w: 2, wtimeout: 1000 }, commandResult.writeConcern);
+            db.collection('test').dropIndexes(
```

Comment:
are these all just whitespace changes? If so, let's try to keep those relegated to separate commits/PRs